sympy/functions/combinatorial/numbers.py

```diff
@@ -2085,3 +2085,23 @@ def find_first_n_motzkin(n):
             motzkin.append(next_num)
             i = i + 1
         return motzkin
+
+    def eval_motzkin(n):
+        if (n-(int(n)))!=0 or n<=0:
```

Comment:
```
n = as_int(n)  # this will raise error if it's not an int -- just leave that to SymPy
if n <= 0:
    raise ValueError('The number must be positive.')
```

Reply:
okay i will get to it.